server/routes/stock-details.js

```diff
@@ -0,0 +1,56 @@
+var express = require('express');
+var router = express.Router();
+
+const defaultDetails = {
+    company_name: "",
+    symbol: "",
+    previous_close: 0,
+    open: 0,
+    last_price: 0,
+    change: 0,
+    change_percent: "",
+    currency: "",
+    volume: "",
+    shares_owned: 0,
+    avg_vol: "",
+    market_cap: "",
+}
+
+const stockInfo = [
+    {
+        company_name: "Tesla",
+        symbol: "TSLA",
+        previous_close: 570.15,
+        open: 602.14,
+        last_price: 609.89,
+        change: -0.23,
+        change_percent: "-0.04%",
```

Comment:
I think we want to store all these metrics as floats rather than strings, so we can manipulate them more easily (i.e., once we get to the customizing stage), and then just concatenate the missing unit symbol (e.g. %, M, B, etc.)

This applies to:
-change_percent
-volume
-avg_vol
-market_cap

And then we'd have to change the corresponding default values for those keys as well, from `""` to `0`

Reply:
Good catch.